app/src/main/kotlin/io/element/android/x/di/AppModule.kt

```diff
@@ -73,23 +73,26 @@ object AppModule {
         @ApplicationContext context: Context,
         buildType: BuildType,
         enterpriseService: EnterpriseService,
-    ) = BuildMeta(
-        isDebuggable = BuildConfig.DEBUG,
-        buildType = buildType,
-        applicationName = ApplicationConfig.APPLICATION_NAME.takeIf { it.isNotEmpty() } ?: context.getString(R.string.app_name),
-        productionApplicationName = ApplicationConfig.PRODUCTION_APPLICATION_NAME,
-        desktopApplicationName = ApplicationConfig.DESKTOP_APPLICATION_NAME,
-        applicationId = BuildConfig.APPLICATION_ID,
-        isEnterpriseBuild = enterpriseService.isEnterpriseBuild,
-        // TODO EAx Config.LOW_PRIVACY_LOG_ENABLE,
-        lowPrivacyLoggingEnabled = false,
-        versionName = BuildConfig.VERSION_NAME,
-        versionCode = context.getVersionCodeFromManifest(),
-        gitRevision = BuildConfig.GIT_REVISION,
-        gitBranchName = BuildConfig.GIT_BRANCH_NAME,
-        flavorDescription = BuildConfig.FLAVOR_DESCRIPTION,
-        flavorShortDescription = BuildConfig.SHORT_FLAVOR_DESCRIPTION,
-    )
+    ): BuildMeta {
+        val applicationName = ApplicationConfig.APPLICATION_NAME.takeIf { it.isNotEmpty() } ?: context.getString(R.string.app_name)
```

Comment:
Maybe we should have different variants for desktop and production app names for the enterprise case too?

Reply:
Maybe. We will update the code if this is a new requirement. FTR iOS did the same here: https://github.com/element-hq/element-enterprise/blob/d9c2d058d901872d3a0e8ea55e5f3ff806836c37/Templates/iOS/XcodeGen.pkl#L10-L11